### PR TITLE
Editorial: A couple fixes from OrdinaryFunctionCreate and undefined arguments

### PR DESCRIPTION
**Pull request by devsnek**

### CODE REVIEW

**Comment by bakkot** [comment]
`%AsyncGenerator%` and `%AsyncGeneratorFunction.prototype%` name the same object, per [the table of well-known intrinsics](https://tc39.es/ecma262/#table-7). If we decide we want to change these uses, presumably we should also change the uses in [CreateDynamicFunction](https://tc39.es/ecma262/#sec-createdynamicfunction).

**Comment by devsnek** [comment]
I did change the use in CreateDynamicFunction.

**Comment by bakkot** [comment]
Hah, so you did.

Is there a reason you only changed the `%AsyncGenerator%` case, and not also the `%Generator%` case?

(Also, you might want to split the two parts of this PR - the missed cases from #1411 won't require as much discussion as the editorial change to how `%AsyncGenerator%` is referred to.)

**Comment by shvaikalesh** [comment]
A few more "not present" => `undefined` cases with `littleEndian` in:
* DataView.prototype.getBigInt64
* DataView.prototype.getBigUint64
* DataView.prototype.setBigInt64
* DataView.prototype.setBigUint64

**Comment by devsnek** [comment]
@bakkot %Generator% has no equiv in the table at the moment.

**Comment by bakkot** [comment]
@devsnek Do you just mean that the description for `%Generator%` just says

> The initial value of the "prototype" data property of %GeneratorFunction%

and does not continue

> i.e., %GeneratorFunction.prototype%

? There's no reason that couldn't be added, if that's the only concern.

**Comment by devsnek** [comment]
@bakkot yeah, I would just do it in a separate pr

**Comment by bakkot** [comment]
I guess I feel like it is both a.) weird to mix the change to `%AsyncGenerator%` in with the followup to #1411 and b.) weird to change `%AsyncGenerator%` without also changing `%Generator%`. So, maybe a separate PR with both of those things would be appropriate, if you want to make this change?

File: spec.html [hunk 1]
```diff
@@ -20390,7 +20390,7 @@ <h1>Runtime Semantics: InstantiateFunctionObject</h1>
       </emu-grammar>
       <emu-alg>
         1. Let _name_ be StringValue of |BindingIdentifier|.
-        1. Let _F_ be ! OrdinaryFunctionCreate(%AsyncGenerator%, |FormalParameters|, |AsyncGeneratorBody|, ~non-lexical-this~, _scope_).
+        1. Let _F_ be ! OrdinaryFunctionCreate(%AsyncGeneratorFunction.prototype%, |FormalParameters|, |AsyncGeneratorBody|, ~non-lexical-this~, _scope_).
```
Comment (ljharb) [inline spec.html]: i see this as an equivalent and editorial choice, unless we're trying to get rid of `%AsyncGenerator%` entirely - which we could do if no other specs reference it.

File: spec.html [hunk 2]
```diff
@@ -26993,13 +26993,12 @@ <h1>BigInt.prototype.toLocaleString ( [ _reserved1_ [ , _reserved2_ ] ] )</h1>
       <emu-clause id="sec-bigint.prototype.tostring">
         <h1>BigInt.prototype.toString ( [ _radix_ ] )</h1>
         <emu-note>
-          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ not present or is *undefined* the Number 10 is used as the value of _radix_.</p>
+          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is *undefined* the Number 10 is used as the value of _radix_.</p>
```
Comment (ljharb) [inline spec.html]: i'd prefer to separate out this kind of change from the intrinsic notation change, if possible.

**Comment by bakkot** [comment]
I force-pushed this down to just the not present -> undefined change, and opened https://github.com/tc39/ecma262/issues/2023 to discuss the editorial convention for referring to intrinsics.

**Pull request merged**